### PR TITLE
Force focus on displayName to prevent visual diff flakes in learner context bar

### PR DESCRIPTION
**Pull request by annabelsegalld2l**
Will re-run the visual diff tests a couple of times to ensure the flake is removed. I also ran around 50 times locally with no flaking so I am pretty confident the solution works.

### CODE REVIEW

File: test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.js [hunk 1]
```diff
@@ -36,6 +36,8 @@ describe('learner-context-bar', () => {
 			const listener = new Promise((resolve) => {
 				card.addEventListener('d2l-labs-user-profile-card-opened', resolve, { once: true });
 			});
+			// force focus on the name to prevent diff flakes
+			elem._focusName = true;
```
Comment (annabelsegalld2l) [inline test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.js:40]: I did the same for the user-profile-card tests and they stopped flaking. The underlying issue is a race condition where sometimes the focus does not enter the focus-trap. Setting this private property (only used for tests) ensures the focus will be put on displayName every time.

File: test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.js [hunk 2]
```diff
@@ -36,6 +36,8 @@ describe('learner-context-bar', () => {
 			const listener = new Promise((resolve) => {
 				card.addEventListener('d2l-labs-user-profile-card-opened', resolve, { once: true });
 			});
+			// force focus on the name to prevent diff flakes
+			card._focusName = true;
```
Comment (annabelsegalld2l) [inline test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.js:40]: I did the same for the user-profile-card tests and they stopped flaking. The underlying issue is a race condition where sometimes the focus does not enter the focus-trap. Setting this private property (only used for tests) ensures the focus will be put on displayName every time.

**Review by vix633 — APPROVED** [review]
This is great, thank you! :shipit:

**Review by jeroach — APPROVED** [review]
Thanks so much!

**Pull request merged**